locales/sk/json.json

```diff
@@ -351,37 +351,37 @@
     "June": "Jún",
     "Kazakhstan": "Kazachstan",
     "Kenya": "Keňa",
-    "Key": "Tlačidlo",
+    "Key": "Kľúč",
     "Kiribati": "Kiribati",
     "Korea": "Južná Kórea",
     "Korea, Democratic People's Republic of": "Severná Kórea",
     "Korea, Republic of": "Kórea, Republika",
-    "Kosovo": "Kosovský",
+    "Kosovo": "Kosovo",
```

Comment:
The name also needs to be added to the `_excludes.json` file, otherwise the status will show up as requiring translation.

Reply:
Added